### DISCUSSION TITLE
Json schema error

### DISCUSSION THREAD

**Discussion by AdamOndrejkovic** [json-schema-org/community#110] [Q&A] [answered]
Hi, I get this error but in my schema I have oneOf and it should be array or object. So why do I get an error?

` instancePath: '/form/steps/0/fields/2/defaultValue',
    schemaPath: 'fields_schema_new/properties/defaultValue/anyOf/1/type',
    keyword: 'type',
    params: { type: 'object' },
    message: 'must be object'
`

**Comment by karenetheridge** [comment, marked as answer]
hard to say without seeing the schema or the data :)

**Reply by AdamOndrejkovic** [reply]
I've resolved the issue thank you.
